common/persistence/nosql/nosqlConfigStore.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/uber/cadence/common/config"
 	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/persistence"
+	p "github.com/uber/cadence/common/persistence"
```

Comment:
Why did you add the same import a second time?

Reply:
I didn't see the previous one.